Data.Access.Layer/Models/User.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Access.Layer.Models
+{
+    public class User
+    {
+        public Guid Id { get; set; }
+
+        public string? Name { get; set; }
+
+        public string? Lastname { get; set; }
+
+        public string? Email { get; set; }
+
+        public string? Password { get; set; }
+
+        public Enums.UserRole Role { get; set; }
```

Comment:
Would these pieces of information be enough for a single user?  
How would you handle deleted users? What would happen if we needed to soft delete the user?
Have you check with client do you need to have information about user changes?